Library/Homebrew/cmd/casks.sh

```diff
@@ -13,8 +13,11 @@ homebrew-casks() {
   if [[ -z "${HOMEBREW_NO_INSTALL_FROM_API}" &&
         -f "${HOMEBREW_CACHE}/api/cask_names.txt" ]]
   then
-    cat "${HOMEBREW_CACHE}/api/cask_names.txt"
-    echo
+    {
+      cat "${HOMEBREW_CACHE}/api/cask_names.txt"
+      echo
```

Comment:
The `echo` is necessary because otherwise the last line of `cask_names.txt` gets merged with the first line of the output of `homebrew-items`, creating an invalid cask name. During testing I found that `zy-player` and `0-ad` became the nonexistent `zy-player0-ad`.

Reply:
That makes sense. Thanks for checking.